spec/factories/card.rb

```diff
@@ -3,6 +3,6 @@
     original_text "How are you?"
     translated_text "Kamusta?"
     review_date DateTime.now.getutc
-    user
+    user_id 1
```

Comment:
@kennyfrc use sequences to solve this kind of problems:
https://github.com/thoughtbot/factory_girl/blob/master/GETTING_STARTED.md#sequences